README.md

```diff
@@ -510,6 +510,10 @@ Path to your `recovery.conf` file.
 
 Sets the proxy option for the official PostgreSQL yum-repositories only. Debian is currently not supported. This is useful if your server is behind a corporate firewall and needs to use proxy servers for outside connectivity.
 
+##### `repo_baseurl`
+
+Sets the baseurl for the PostgreSQL repository. Useful if you host your own mirror of the repository.
```

Comment:
Maybe mention the default will be the official PostgreSQL repos for apt and yum depending on OS? I wouldn't say what they are, just leave that vague perhaps to save future unnecessary edits.